### ISSUE TITLE
NoMethodError (undefined method `numero' for "Um Milhão":String)

### ISSUE THREAD

**Issue by diegopolido** [rickmax/extensobr#4]
### WHY

```
2.6.5 :002 > Extenso.moeda(100000 * 100)
 => "Cem mil Reais" 
2.6.5 :003 > Extenso.moeda(1000000 * 100)
Traceback (most recent call last):
        1: from (irb):3
NoMethodError (undefined method `numero' for "Um Milhão":String)

2.6.5 :004 > Extenso.numero(1000000)
Traceback (most recent call last):
        2: from (irb):4
        1: from (irb):4:in `rescue in irb_binding'
NoMethodError (undefined method `numero' for "Um Milhão":String)
```

Generated by Ruby on Rails application

```
$ ruby -v
ruby 2.6.5p114 (2019-10-01 revision 67812) [x86_64-linux]
$ rails -v
Rails 5.2.4.1
```

**Comment by diegopolido** [comment]
I also noticed that "Um milhão Reais" should be "Um milhão de Reais"

In the gem console didn't throw the error:

```
~/Projects/extensobr$ ruby bin/console 
2.6.3 :001 > Extenso.numero(1000000)
 => "Um Milhão" 
2.6.3 :002 > Extenso.moeda(1000000 * 100)
 => "Um Milhão Reais" 
2.6.3 :003 > exit
```

**Comment by diegopolido** [comment]
Maybe it's some conflict with Rails somewhere?

**Comment by rickmax** [comment]
Hi,

Thanks for the notifications, all issues have already been fixed, update your gem to 'extensobr', '~> 0.1.2'.

Att
Henrique Max
Software Development
rickmaxg3@hotmail.com
